target/linux/x86/Makefile

```diff
@@ -23,6 +23,9 @@ DEFAULT_PACKAGES += partx-utils mkf2fs e2fsprogs
 
 $(eval $(call BuildTarget))
 
-$(eval $(call $(if $(CONFIG_ISO_IMAGES),RequireCommand,Ignore),mkisofs, \
-   	Please install mkisofs. \
+$(eval $(call $(if $(CONFIG_ISO_IMAGES),SetupHostCommand,Ignore),mkisofs, \
+	Please install mkisofs. , \
```

Comment:
Why is this change needed? This should probably go into separate commit.

Reply:
Yes, it should go into separate commit. This change just because my build environment (debian) provide genisoimage instead mkisofs.

Reply:
Ok, so I'll remove it, feel free to do it in separate PR/patch later.